LDAR_Sim/src/programs/component_level_method.py

```diff
@@ -204,4 +202,10 @@ def deploy_crews(self, workplan: Workplan, weather, daylight) -> CrewDeploymentS
             workplan.add_survey_report(survey_report, survey_plan)
```

Comment:
Problem: Should add the survey reports to the workplan after the loop here like we are doing in method.py - Perhaps a copy of the unit tests for method.py needs to be transferred and exist for component_level_method as well in order to catch things like this